src/packages/limitations.md

```diff
@@ -2,37 +2,38 @@
 
 When using or even considering third-party software delivered with CheriBSD
 packages, it is important to keep in mind the maturity of currently available
-CheriABI packages and potential conflicts between hybrid ABI and CheriABI
+CheriABI packages, and potential conflicts between hybrid ABI and CheriABI
 packages.
 The following sections describe some of such limitations.
 
 ## Maturity of CheriABI packages
 
 There are many projects that must still be ported to CHERI in order to provide
-a fully functional CheriABI package repository.
+a fully-functional CheriABI package repository.
 The available CheriABI packages are also not continuously tested as part of the
 CheriBSD package building infrastructure.
 
-CheriABI packages compile, but many have CHERI-related warnings that
-have not been audited and only a limited set have been tested.
-While the CheriABI packages are more interesting as they use CHERI memory-safety
-features, you must remember that they might break in run-time and hence might
-not be suitable for critical operations.
-In such case, a corresponding hybrid ABI package might be considered instead of
-a CheriABI package.
+CheriABI packages compile, but many have CHERI-related warnings that have not
+been audited, and only a limited set have been tested.
+Whilst the CheriABI packages are more interesting as they use CHERI
+memory-safety features, they might break at run time and hence might not be
+suitable for critical operations.
+In such cases, a corresponding hybrid ABI package might be better suited than
+the CheriABI package.
 
 We expect the number of available and tested packages to grow with future
-CheriBSD releases but it will take quite some time until we will be able to
-provide a CheriABI package repository that would include all packages currently
-provided with the hybrid ABI package repository.
+CheriBSD releases, but it will take quite some time until we are able to provide
+a CheriABI package repository that includes all packages currently provided in
+the hybrid ABI package repository.
```

Comment:
"all packages ..." probably isn't a goal. Not sure how best to temper that or if we should bother.

Reply:
Maybe more broadly like this (in the current PR version)?
> (..) provide a CheriABI package repository that covers the same functionalities which are included in the hybrid ABI package repository as of today.

Reply:
works for me